CentralHub.Api.Tests/MockAggregateRepository.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections.Immutable;
+using CentralHub.Api.Dtos;
+using CentralHub.Api.Model;
+using CentralHub.Api.Services;
+
+namespace CentralHub.Api.Tests;
+
+public class MockAggregateRepository : IAggregatedMeasurementRepository
+{
+    private readonly Dictionary<int, List<MeasurementGroup>> _recentMeasurementGroups = new Dictionary<int, List<MeasurementGroup>>();
```

Comment:
This should be static and use a lock the same with the other thing. We should be able to use this mock repository in our real code and right now we cant because it will get an unique mock repository with its own data for each request.